Xamarin.Forms.Core/DispatcherExtensions.cs

```diff
@@ -26,7 +26,14 @@ public static IDispatcher GetDispatcher(this BindableObject bindableObject)
 			}
 
 			// Use the DispatcherProvider to retrieve an appropriate dispatcher for this BindableObject
-			return s_current.GetDispatcher(bindableObject);
+			IDispatcher dispatcher = s_current.GetDispatcher(bindableObject);
```

Comment:
```suggestion
			return s_current.GetDispatcher(bindableObject) ?? new FallbackDispatcher();
```

Reply:
What should I do now? Commit or batch? Thank you for your advice.

Reply:
@TomOnd either way

When we merge things in we squash all the commits down to one so it doesn't really matter

Reply:
Resolved with commit: a1b0093